### PR TITLE
nvidia: update to 495.46.

### PR DESCRIPTION
**Pull request by motorto**
#### Testing the changes
- I tested the changes in this PR: **NO**

- I built this PR locally for my native architecture, x86_64

Still haven't got the time to test on hardware, but it builds and installs fine .

Testers are welcome, hope I didn't miss anything  ...

@abenson  how does it look for you ?

### CODE REVIEW

**Comment by abenson** [comment]
1. We typically stick to Production branch releases for `nvidia`.
2. This is a good start for what will be the next `nvidia` major update, so keep it open.
3. The current 470.xx branch will need to forked off to `nvidia470` since 495.xx drops support for some generations of hardware.

[This is my WIP](https://github.com/abenson/void-packages/commit/2fdd8261dff686a60b349299cba99d7329919ba6).  It looks like you're missing a few pieces, specifically [nvidia-drm_gbm.so](https://github.com/abenson/void-packages/commit/2fdd8261dff686a60b349299cba99d7329919ba6#diff-c21b8dffa622241f09abbac304b9eecbadf551b779bb8c21e191e45ab62c783aR103) and [15_nvidia_gbm.json](https://github.com/abenson/void-packages/commit/2fdd8261dff686a60b349299cba99d7329919ba6#diff-c21b8dffa622241f09abbac304b9eecbadf551b779bb8c21e191e45ab62c783aR260)

**Comment by motorto** [comment]
Sorry, wasn't aware of your wip already. I Should have asked in the irc. 

I committed adding the missing parts, I will squash them later ... Though now for some reason is failing the lint check.

**Comment by abenson** [comment]
> Though now for some reason is failing the lint check.

That'll be fixed if you squash the commits.  It's complaining about the "add missing parts" as a commit subject.

**Comment by motorto** [comment]
> The current 470.xx branch will need to forked off to nvidia470 since 495.xx drops support for some generations of hardware.

Is there a quick way to do this ? maybe some git command that I am not aware of ? Or we need to manually create the respective files ?

**Comment by abenson** [comment]
Copy the old `nvidia` to `nvidia470`, then update the references in the template to `nvidia470`.

**Comment by motorto** [comment]
Could you check that I did it right ?  got a little confused with the symbolic links not sure if I had to create them ...

File: srcpkgs/nvidia470/files/dkms.conf [hunk 1]
```diff
@@ -0,0 +1,19 @@
+PACKAGE_NAME="nvidia"
```
Comment (abenson) [inline srcpkgs/nvidia470/files/dkms.conf:1]: Looking at `nvidia390`, this should be `nvidia470`

File: srcpkgs/nvidia470/template [hunk 2]
```diff
@@ -0,0 +1,311 @@
+# Template file for 'nvidia470'
+
+_desc="nvidia470 drivers for linux"
```
Comment (abenson) [inline srcpkgs/nvidia470/template:3]: To match `nvidia390`, maybe:
 
    _desc="NVIDIA drivers (GeForce 600, most 700 series)"

**Comment by KSPAtlas** [comment]
Any progress? I really need these drivers.

**Comment by wael444** [comment]
> Any progress? I really need these drivers.

https://github.com/void-linux/void-packages/pull/34911

**Pull request closed without merging**